### PR TITLE
website/docs: release notes for 2024.6

### PR DESCRIPTION
**Pull request by BeryJu**
## Details

REPLACE ME

---

## Checklist

-   [ ] Local tests pass (`ak test authentik/`)
-   [ ] The code has been formatted (`make lint-fix`)

If an API change has been made

-   [ ] The API schema has been updated (`make gen-build`)

If changes to the frontend have been made

-   [ ] The code has been formatted (`make web`)

If applicable

-   [ ] The documentation has been updated
-   [ ] The documentation has been formatted (`make website`)

### CODE REVIEW

**Comment by rissson** [comment]
Updated with the changes from #9727

File: website/docs/releases/2024/v2024.6.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+To try out the release candidate, replace your Docker image tag with the latest release candidate number, such as 2024.6.0-rc1. You can find the latest one in [the latest releases on GitHub](https://github.com/goauthentik/authentik/releases). If you don't find any, it means we haven't released one yet.
+:::::
+
+## Breaking changes
+
+### PostgreSQL minimum supported version upgrade
+
+authentik now requires PostgreSQL version 14 or later. We recommend upgrading to the latest version if you are running an older version.
+
+The provided Helm chart defaults to PostgreSQL 15. If you are using the Helm chart with the default values, no action is required.
```
Comment (tanberry) [inline website/docs/releases/2024/v2024.6.md:18]: Even if one is running PostgreSQL v14, it's ok that the Helm chart wants 15?

File: website/docs/releases/2024/v2024.6.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+2024.6 has not been released yet! We're publishing these release notes as a preview of what's to come, and for our awesome beta testers trying out release candidates.
+
+To try out the release candidate, replace your Docker image tag with the latest release candidate number, such as 2024.6.0-rc1. You can find the latest one in [the latest releases on GitHub](https://github.com/goauthentik/authentik/releases). If you don't find any, it means we haven't released one yet.
+:::::
+
+## Breaking changes
+
+### PostgreSQL minimum supported version upgrade
+
+authentik now requires PostgreSQL version 14 or later. We recommend upgrading to the latest version if you are running an older version.
```
Comment (tanberry) [inline website/docs/releases/2024/v2024.6.md:16]:
```suggestion
With this release, authentik now requires PostgreSQL version 14 or later. We recommend upgrading to the latest version if you are running an older version.
```

File: website/docs/releases/2024/v2024.6.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+The provided Helm chart defaults to PostgreSQL 15. If you are using the Helm chart with the default values, no action is required.
+
+The provided Compose file was updated with PostgreSQL 16. You can follow the procedure [here](../../troubleshooting/postgres/upgrade_docker.md) to upgrade.
+
+## New features
+
+-   Google Workspace Provider (preview) (enterprise)
```
Comment (tanberry) [inline website/docs/releases/2024/v2024.6.md:24]:
```suggestion
-   Google Workspace Provider (preview) (Enterprise)
```

File: website/docs/releases/2024/v2024.6.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+-   Microsoft Entra Provider (preview) (enterprise)
```
Comment (tanberry) [inline website/docs/releases/2024/v2024.6.md:25]:
```suggestion
-   Microsoft Entra Provider (preview) (Enterprise)
```

File: website/docs/releases/2024/v2024.6.md [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+-   Reworked proxy provider redirect
+-   Optimize sync and property mapping execution
+-   Read-replica DB support
+
+## Upgrading
+
+authentik now requires PostgreSQL version 14 or later. We recommend upgrading to the latest version if needed. Follow the instructions [here](../../troubleshooting/postgres/upgrade_docker.md) if you need to upgrade PostgreSQL with docker-compose.
```
Comment (tanberry) [inline website/docs/releases/2024/v2024.6.md:32]:
```suggestion
With this release, authentik now requires PostgreSQL version 14 or later. We recommend upgrading to the latest version if needed. Follow the instructions [here](../../troubleshooting/postgres/upgrade_docker.md) if you need to upgrade PostgreSQL with docker-compose.
```

File: website/docs/releases/2024/v2024.6.md [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+
+### docker-compose
```
Comment (tanberry) [inline website/docs/releases/2024/v2024.6.md:34]:
```suggestion
### Docker Compose
```

File: website/docs/releases/2024/v2024.6.md [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+
+To upgrade, download the new docker-compose file and update the Docker stack with the new version, using these commands:
```
Comment (tanberry) [inline website/docs/releases/2024/v2024.6.md:36]:
```suggestion
To upgrade, download the new `docker-compose.yml` file and update the Docker stack with the new version, using these commands:
```

File: website/docs/releases/2024/v2024.6.md [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+-   enterprise/providers/microsoft_entra: initial account sync to microsoft entra (#9632)
+-   enterprise/providers/microsoft_entra: fix error when updating connection attributes (#10039)
+-   enterprise/stages/source: fix error when creating source stage from blueprint (#9810)
+-   events: ensure all models' **str** can be called without any further lookups (#9480)
+-   events: fix geoip enrich context not converting to json-seriazable data (#9885)
+-   flows: fix execute API endpoint (#9478)
+-   lib/providers/sync: improve outgoing sync (#9835)
+-   lib/providers/sync: multiple minor fixes (#9667)
+-   lib/sync/outgoing, sources/ldap: fix sync status endpoint (#9855)
+-   lib/sync: fix json error (#9827)
```
Comment (tanberry) [inline website/docs/releases/2024/v2024.6.md:81]:
```suggestion
-   lib/sync: fix JSON error (#9827)
```

File: website/docs/releases/2024/v2024.6.md [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+-   core: only prefetch related objects when required (#9476)
+-   crypto: update fingerprint at same time as certificate (#10036)
+-   enterprise/audit: fix audit logging with m2m relations (#9571)
+-   enterprise/providers/google: initial account sync to google workspace (#9384)
+-   enterprise/providers/google_workspace: ensure no group is created when no property mappings are set (#9783)
+-   enterprise/providers/microsoft_entra: initial account sync to microsoft entra (#9632)
+-   enterprise/providers/microsoft_entra: fix error when updating connection attributes (#10039)
+-   enterprise/stages/source: fix error when creating source stage from blueprint (#9810)
+-   events: ensure all models' **str** can be called without any further lookups (#9480)
+-   events: fix geoip enrich context not converting to json-seriazable data (#9885)
```
Comment (tanberry) [inline website/docs/releases/2024/v2024.6.md:76]:
```suggestion
-   events: fix GeoIP enrich context not converting to json-seriazable data (#9885)
```

File: website/docs/releases/2024/v2024.6.md [hunk 10]
```diff
@@ -0,0 +1,138 @@
...
+-   lib/providers/sync: improve outgoing sync (#9835)
+-   lib/providers/sync: multiple minor fixes (#9667)
+-   lib/sync/outgoing, sources/ldap: fix sync status endpoint (#9855)
+-   lib/sync: fix json error (#9827)
+-   lifecycle/migrate: only acquire lock once (#9856)
+-   lifecycle: always try custom redis URL (#9441)
+-   lifecycle: close database connection after migrating (#9516)
+-   lifecycle: fix ak test-all command
+-   policies: fix ak_call_policy failing when used in testing (#9853)
+-   providers/oauth2: don't handle api scope as special scope (#9910)
```
Comment (tanberry) [inline website/docs/releases/2024/v2024.6.md:87]:
```suggestion
-   providers/oauth2: don't handle API scope as special scope (#9910)
```

File: website/sidebars.js [hunk 11]
```diff
@@ -412,13 +412,14 @@ const docsSidebar = {
                 description: "Release notes for recent authentik versions",
```
Comment (tanberry) [inline website/sidebars.js:412]:
```suggestion
                description: "Release Notes for recent authentik versions",
```

**Review by tanberry — APPROVED** [review]
Some nits and a question. :-)

**Pull request merged**